managed-databases/postgresql-and-mysql/how-to/create-read-replica.mdx

```diff
@@ -0,0 +1,53 @@
+---
+meta:
+  title: How to create read replicas
+  description: This page explains how to create read replicas
+content:
+  h1: How to create read replicas
+  paragraph: This page explains how to create read replicas
+dates:
+  validation: 2022-08-31
+  posted: 2019-09-21
+---
+
+A read replica is a live copy of a Database Instance that behaves like an Instance, but that only allows read-only connections. The replica mirrors the data of the primary Database Instance and any changes made are synchronized and reflected in the replica.
+
+Read replicas allow you to scale your Database Instance for read-heavy database workloads. They can also be used for business intelligence workloads. 
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+- You have a [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart)
+
+</Message>
+
+## How to create a read replica
+
+1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
+3. Scroll down to the **Read Replica** section. 
+4. Click **Create Read Replica**. A pop-up displays.
+5. Select the network type. Two types are available:
+
+    - **Public Network** - Resources in a public network are publicly visible by default. The default Access Control List (ACL) settings allow anybody to connect to resources in your network. You can restrict access by configuring ACLs.
+    - **Private Networks** - Private Networks allow you to create virtual Private Networks for your Databases, enabling them to communicate in an isolated network without the need for a public IP address. If you select this option, follow these additional steps:
+      - Select an existing Private Network to attach to your read replica.
+      - Configure your IP address and its corresponding mask. 
+
+  <Message type="note">
+
+  Read replicas can have at most one direct access and one Private Network endpoint.
+
```

Comment:
Maybe we can add a N.B. saying that if the user choses public network at this level, he can still attach a private network to his read replica afterwards. 
If he choses private network, he will not be able to add a public network to the same read replica.